db/migrate/20230519144435_add_application_status_to_course.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddApplicationStatusToCourse < ActiveRecord::Migration[7.0]
+  def change
+    add_column :course, :application_status, :integer, default: 0, null: false
```

Comment:
let's an index here, we'll definitely be searching against this (from Find)

Reply:
Yup, done: https://github.com/DFE-Digital/publish-teacher-training/pull/3578/commits/8f1e90b28533a256f853ab2c6886368838abed90